dapps/multisig-toolkit/src/components/header.tsx

```diff
@@ -9,6 +9,7 @@ const links = [
 	{ to: '/signature-analyzer', label: 'Signature Analyzer' },
 	{ to: '/multisig-address', label: 'MultiSig Address' },
 	{ to: '/combine-signatures', label: 'Combine MultiSig Signatures' },
+	{ to: '/broadcast-transaction', label: 'Broadcast Transaction'},
```

Comment:
Can we call this `execute-transaction` just to be consistent with terminology everywhere?